site/content/en/docs/formats/kitti.md

```diff
@@ -48,6 +48,8 @@ KITTI segmentation dataset directory should have the following structure:
 <!--lint disable fenced-code-flag-->
 ```
 └─ Dataset/
+    ├── label_colors.txt # optional, color map for non-original segmentation labels
+    ├── labels.txt # optional, file for non-original detection labels
```

Comment:
Can we use only one file?

Reply:
Yes, we can, but the format support the ability to have different labels for detection and segmentation tasks

Reply:
What happens, if there is 2 files?